VTEX - Marketplace Protocol - External Marketplace Mapper.json

```diff
@@ -405,335 +592,252 @@
-                                    "isActive",
-                                    "helpUrl",
-                                    "connectorVtexAdminPath",
-                                    "connectorPageUrl",
-                                    "connectorApiEndpoint",
-                                    "channelId"
-                                ],
                                 "properties": {
                                     "name": {
                                         "type": "string",
-                                        "title": "name",
+                                        "description": "Connector name.",
                                         "example": "Mercado Livre Classic"
                                     },
                                     "website": {
                                         "type": "string",
-                                        "title": "website",
+                                        "description": "Connector website.",
                                         "example": "mercadolivre.com.br"
                                     },
                                     "logo": {
                                         "type": "string",
-                                        "title": "logo",
+                                        "description": "Connector logo URL.",
                                         "example": "https://g.foolcdn.com/art/companylogos/square/meli.png"
                                     },
                                     "description": {
                                         "type": "string",
-                                        "title": "description",
+                                        "description": "Connector description.",
                                         "example": "MercadoLibre operates online commerce platforms in Latin America. It operates Mercado Libre Marketplace, an automated online commerce platform that enables businesses, merchants, and individuals to list merchandise and conduct sales and purchases online."
                                     },
                                     "segment": {
                                         "type": "string",
-                                        "title": "segment",
-                                        "example": "OTHERS"
+                                        "description": "Market segment.",
+                                        "example": "OTHERS",
                                     },
                                     "availableCountries": {
+                                        "description": "List of countries where connector is available for integration.",
                                         "type": "array",
-                                        "title": "availableCountries",
                                         "items": {
+                                            "description": "Country code.",
```

Comment:
Sugiro uma descrição mais completa, por exemplo:

"Three-digit country code, in [ISO 3166 ALPHA-3](https://www.iban.com/country-codes) format."